examples/counter-metrics/counter_metrics.server.out

```diff
@@ -1,9 +1,9 @@
 # To start the service, navigate to the directory that contains the
-# `.bal` file and execute the `bal run` command below with the build time flag `--observability-included` and the runtime flag `--b7a.observability.metrics.enabled=true`.
-bal run --observability-included counter_metrics.bal --b7a.observability.metrics.enabled=true
+# `.bal` file and execute the `bal run` command below with the build time flag `--observability-included` and the runtime configuration file `Config.toml`.
```

Comment:
```suggestion
# `.bal` file and execute the `bal run` command below with the `--observability-included` build time flag and the `Config.toml` runtime configuration file.
```
Please correct in all occurrences.

Reply:
Fixed with 3c257fc977bdca22f195e0a54e89aff023695e39